.ado/templates/android-dep-setup.yml

```diff
@@ -0,0 +1,5 @@
+steps:
+  - task: Bash@3
+    inputs:
+      targetType: 'inline'
+      script: echo "y" | $ANDROID_HOME/tools/bin/sdkmanager --install 'ndk;21.0.6113669'
```

Comment:
Is there not an equivalent azure pipeline task to do this?

Reply:
default Agents come with specific NDK version 22.x.x, which does not fit for latest android gradle as well.
There are no azure tasks I could find to download them as well.